runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -6607,32 +6560,12 @@ done:;
 	VMINLINE VM_BytecodeAction
 	invokestaticsplit(REGISTER_ARGS_LIST)
 	{
-		VM_BytecodeAction rc = GOTO_RUN_METHOD;
 		U_16 splitTableIndex = *(U_16*)(_pc + 1);
+		profileCallingMethod(REGISTER_ARGS);
```

Comment:
Can this result in profiling the initialStaticMethod?  Is the iprofiler ready to handle that?

Reply:
The profile here is for the calling method, not the target method. invokespecial already did this.

```	VMINLINE void
	profileCallingMethod(REGISTER_ARGS_LIST)
	{
		U_8 *profilingCursor = startProfilingRecord(REGISTER_ARGS, sizeof(J9Method*));
		if (NULL != profilingCursor) {
			*(J9Method**)profilingCursor = _literals;
		}
	}
```

Reply:
invokestatic does exactly this.